### PR TITLE
Authentication configuration guide

### PR DESCRIPTION
**Pull request by michaelstonis**
Provides a guide for configuring authentication in the API services.

### CODE REVIEW

File: docs/demos/authentication/README.md [hunk 1]
```diff
@@ -0,0 +1,113 @@
...
+       "PodcastDb": "Server=localhost, 5433;Database=Podcast;User Id=sa;Password=Pass@word;Encrypt=False",
+       "FeedQueue": "UseDevelopmentStorage=true"
+     },
+     "AzureAd": {
+       "Instance": "https://login.microsoftonline.com/",
+       "Domain": "{DOMAIN}",
+       "TenantId": "{AZURE_AD_TENANT_ID}",
+       "ClientId": "{AZURE_AD_CLIENT_ID}"
+     },
+     "Authentication": {
```
Comment (captainsafia) [inline docs/demos/authentication/README.md:73]: We may want to remove this config section and show that it gets populated after the user runs `dotnet user-jwts` below.

**Review by captainsafia — APPROVED** [review]
LGTM except for one comment so I'll proactively approve.

File: docs/demos/authentication/README.md (hunk 1 above)
Comment (captainsafia) [reply docs/demos/authentication/README.md:73]: Correct. Just the authentication section needs to be removed.

Comment (jamesmontemagno) [reply docs/demos/authentication/README.md:73]: @michaelstonis any update on this?

**Pull request merged**